docs/docs/examples/core-components/signal-interoperability.md

```diff
@@ -0,0 +1,280 @@
+---
+sidebar_position: 1
+---
+
+import signalToRos from './assets/signal-ros-example.png'
+import rosToSignal from './assets/ros-signal-example.png'
+
+# Signal interoperability
+
+The examples below show how signal translator components included in the core components can be used with a selection of
+common ROS message types for interoperability.
+
+:::tip
+
+This documentation contains a section entirely dedicated to [signals](../../concepts/05-building-blocks/01-signals.md).
+
+:::
+
+## Example 1
+
+The first example uses the `Joint Signal To Joint State Message` component to translate the joint state output from the
+hardware interface to a `sensor_msgs::msg::JointState` message and the `Cartesian Signal to Pose Stamped` component to
+translate the Cartesian state output to a `geometry_msgs::msg::PoseStamped` message.
+
+:::note
+
+Instead of the `Cartesian Signal to Pose Stamped`, one could also use the `Cartesian Signal to Twist Stamped` or
+`Cartesian Signal to Wrench Stamped` to translate to a `geometry_msgs::msg::TwistStamped` or
+`geometry_msgs::msg::WrenchStamped`, respectively.
+
+:::
+
+<div class="text--center">
+  <img src={signalToRos} alt="Signal interoperability example 1" />
+</div>
+
+With the application loaded and playing, the two components will publish the converted message on their output signal
+each time a new message is received from the hardware interface. A jsonified version of those messages can be observed
+in the live topic view.
+
+![signal ros pose](./assets/signal-ros-pose.png)
+
+![signal ros joint](./assets/signal-ros-joint.png)
```

Comment:
These are super small images to fit all the content. I think it would be better if they were "normal" size with the majority of the message being cut-off, and then underneath an inline code block to paste the json output, so that it is automatically formatted with a horizontal scrollbar for the long text.
```console
{"header":{"stamp":{................................................................................................................................................................................................................................................
```

Reply:
![image](https://github.com/user-attachments/assets/9f11f938-e879-4a07-8937-c2f7559fb1be)